service/history/workflowResetter.go

```diff
@@ -365,6 +365,7 @@ func (r *workflowResetterImpl) persistToDB(
 			currentWorkflowEventsSeq,
 			resetWorkflowSnapshot,
 			resetWorkflowEventsSeq,
+			currentWorkflow.getMutableState().GetNamespaceEntry().IsGlobalNamespace(),
```

Comment:
currentWorkflow -> resetWorkflow

Reply:
given that the tests were passing without this correction, doesn't that mean that we don't have test that validates this functionality?